subworkflows/local/hmmclassify.nf

```diff
@@ -0,0 +1,33 @@
+include { HMMER_HMMSEARCH } from '../../modules/nf-core/hmmer/hmmsearch/main'
+include { HMMRANK         } from '../../modules/local/hmmrank'
+
+workflow HMMCLASSIFY {
+
+    take:
+    ch_hmmclassify // channel: [ val(meta), [ hmm, aa_fasta ] ]
+
+    main:
+    ch_versions = Channel.empty()
+
+    HMMER_HMMSEARCH ( 
+        ch_hmmclassify
+            .map { [ [ id: "${it[0].id}.${it[1].baseName}" ], it[1], it[2], false, true, false ] }
```

Comment:
Can you give me an example? I don't understand what should I do 🥴

Reply:
Closures can take named arguments instead of just using the default variable `it`. So in this case you would do ( hopefully this is correct looking at the module )
```suggestion
            .map { meta, hmm, seqdb -> [ [ id: "${meta.id}.${hmm.baseName}" ], hmm, seqdb, false, true, false ] }
```
This way I have a better idea of what is being passed in the channels.